docs/source/getting-started/install.md

```diff
@@ -174,10 +174,21 @@ Use the project generator helper utility.
 1.  Open a terminal and execute:
 
     ```bash
-    $ npm install -g yo @plone/generator-volto
-    $ yo @plone/volto
+    npm install -g yo @plone/generator-volto
+    yo @plone/volto
     ```
 
+      ````{hint}
```

Comment:
The indentation here of the backticks appears to be a little bit off. It should align with the "O" in "Open a terminal and execute:" above, its content, and the closing 4 backticks below.